cis14-enable/enablecis14.py

```diff
@@ -74,7 +74,7 @@ def assume_role(aws_account_number, role_name):
     # Setup command line arguments
     parser = argparse.ArgumentParser(description='Enable CIS 1.4 in Security Hub accounts')
     parser.add_argument('--assume_role', type=str, required=True, help="Role Name to assume in each account.")
-    parser.add_argument('--enabled_regions', type=str, required=True, help="Comma separated list of regions to enable CIS 1.4. If not specified, all available regions enabled.")
+    parser.add_argument('--enabled_regions', type=str, required=False, help="Comma separated list of regions to enable CIS 1.4. If not specified, all available regions enabled.")
```

Comment:
In this if statement check the user set the enabled_regions of args, then if is not set enable all regions. But when enable_regions was required so it doesn't work properly. So I changed to optionally.

```python
    securityhub_regions = []
    if args.enabled_regions:
        securityhub_regions = [str(item) for item in args.enabled_regions.split(',')]
        print("Enabling members in these regions: {}".format(securityhub_regions))
    else:
        securityhub_regions = session.get_available_regions('securityhub') 
        print("Enabling CIS 1.4 in all available SecurityHub regions {}".format(securityhub_regions))

```